Cargo.toml

```diff
@@ -34,6 +34,7 @@ std = []
 no-std = ["no-std-compat/std", "dep:spin"]
 f16 = ["dep:half"]
 ci-check = []
+alloc_async = []
```

Comment:
Is it possible to check this at runtime instead of feature flag?

Reply:
I'm not sure, the error is "Operation not supported". I'm not sure trying alloc and retrying with sync alloc is a good general approach. 

To run on this older card, I definitely need this bit of code, whether it's worth keeping in the library is not is not really clear.
Tbh, if I was maintaining, I wouldn't add this flag to the main library. I'm happy to keep this as a fork, until more demand and or clearer path for support.